MantidQt/MantidWidgets/inc/MantidQtMantidWidgets/DataProcessorUI/GenericDataProcessorPresenter.h

```diff
@@ -109,19 +115,14 @@ class EXPORT_OPT_MANTIDQT_MANTIDWIDGETS GenericDataProcessorPresenter
   DataProcessorMainPresenter *m_mainPresenter;
   // stores whether or not the table has changed since it was last saved
   bool m_tableDirty;
-  // Index for column 'Group'
-  int m_colGroup;
   // stores the user options for the presenter
   std::map<std::string, QVariant> m_options;
   // Post-process some rows
-  void postProcessGroup(int groupId, const std::set<int> &rows);
+  void postProcessGroup(const std::set<std::vector<std::string>> &data);
```

Comment:
The data in the model as returned by the `TreeManager`. Because the presenter only keeps a pointer to the abstract tree manager and is no longer aware of the model (and therefore does not know how to read from/write to it), I needed to find a format, common to both situations (post-processing and no-postprocessing), to store the data from the table that the presenter can understand and use.

The set is the set of rows that belong to the same group (it will be a set of a single element if there are no groups). For each element in the set, we have a vector that contains the values in each column. For instance:

`std::vector<std::string> = {"13463", "0.5", "13464", ...}`

means "Run"=13463, "ThetaIn"=0.5, "TransmissionRun=13464"